lib/runtime.coffee

```diff
@@ -65,11 +65,16 @@ class Runtime
   # Public: stops execution of the current fork
   stop: ->
     @runner.stop()
+    @emitter.emit 'did-execute-stop'
 
   # Public: Dispatched when the execution is starting
   onDidExecuteStart: (callback) ->
     @emitter.on 'did-execute-start', callback
 
+  # Public: Dispatched when the execution is stopped
+  onDidExecuteStop: (callback) ->
+    @emitter.on 'did-execute-stop', callback
```

Comment:
I'd name this `stopped` or `stop` to fit better with node nomenclatures on events, but this works. If you agree, feel free to submit a follow on PR.